docs/user-guide/walkthrough/README.md

```diff
@@ -12,77 +12,176 @@ certainly want the docs that go with that version.</h1>
+
+If your cluster was deployed by other means (e.g. a [getting started guide](../../getting-started-guides/README.md)), you may want to configure the path to the Kubernetes apiserver in your shell environment:
+
+```sh
+export KUBERNETES_MASTER=http://<ip>:<port>/api
+```
+
+Check that kubectl is properly configured by getting the cluster state:
+
+```sh
+kubectl cluster-info
+```
+
 
 ## Pods
-The first atom of Kubernetes is a _pod_.  A pod is a collection of containers that are symbiotically grouped.
+In Kubernetes, a group of one or more containers is called a _pod_. Containers in a pod are deployed together, and are started, stopped, and replicated as a group.
 
 See [pods](../../../docs/user-guide/pods.md) for more details.
 
-### Intro
 
-Trivially, a single container might be a pod.  For example, you can express a simple web server as a pod:
+#### Pod Definition
+
+The simplest pod definition describes the deployment of a single container.  For example, an nginx web server pod might be defined as such:
 
 ```yaml
 apiVersion: v1
 kind: Pod
 metadata:
-  name: www
+  name: nginx
 spec:
   containers:
-    - name: nginx
-      image: nginx
+  - name: nginx
+    image: nginx
+    ports:
+    - containerPort: 80
 ```
 
-A pod definition is a declaration of a _desired state_.  Desired state is a very important concept in the Kubernetes model.  Many things present a desired state to the system, and it is Kubernetes' responsibility to make sure that the current state matches the desired state.  For example, when you create a Pod, you declare that you want the containers in it to be running.  If the containers happen to not be running (e.g. program failure, ...), Kubernetes will continue to (re-)create them for you in order to drive them to the desired state. This process continues until you delete the Pod.
+A pod definition is a declaration of a _desired state_.  Desired state is a very important concept in the Kubernetes model.  Many things present a desired state to the system, and it is Kubernetes' responsibility to make sure that the current state matches the desired state.  For example, when you create a Pod, you declare that you want the containers in it to be running.  If the containers happen to not be running (e.g. program failure, ...), Kubernetes will continue to (re-)create them for you in order to drive them to the desired state. This process continues until the Pod is deleted.
 
 See the [design document](../../../DESIGN.md) for more details.
 
-### Volumes
 
-Now that's great for a static web server, but what about persistent storage?  We know that the container file system only lives as long as the container does, so we need more persistent storage.  To do this, you also declare a ```volume``` as part of your pod, and mount it into a container:
+#### Pod Management
+
+Create a pod containing an nginx server ([pod-nginx.yaml](pod-nginx.yaml)):
+
+```sh
+kubectl create -f examples/walkthrough/pod-nginx.yaml
```

Comment:
The path is docs/user-guide/walkthrough/pod-nginx.yaml not examples/walkthrough/pod-nginx.yaml.

@mikedanese may be fixing this in head.